app/controllers/api/services_controller.rb

```diff
@@ -17,11 +17,11 @@ class Api::ServicesController < Api::BaseController
 
   def index
     activate_menu :dashboard
-    @services = current_user.accessible_services
-                            .order(updated_at: :desc)
-                            .paginate(pagination_params)
-                            .decorate
-                            .to_json(only: %i[name updated_at id system_name], methods: %i[links apps_count backends_count unread_alerts_count])
+    @raw_services = current_user.accessible_services
```

Comment:
If you are not going to use this variable outside this method, better make it a local variable of the method rather than an instance variable of the class that can be used all over the controller, the views and the helpers.

```suggestion
    raw_services = current_user.accessible_services
```

Reply:
I see that you use it for the counter... then better `@total_services_count = ...`
The less 'global variables' with less 'global info', the better.

Reply:
For a following PR 👍